### PR TITLE
service,vendor: update operatorkit

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: service/controller/chart.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/giantswarm/chart-operator/service/controller/v1"
 )
 
+const chartControllerSuffix = "-chart"
```
Comment (kopiczko) [inline service/controller/chart.go]: Note I added this constant, and add this suffix to controller name despite the fact there is only a single controller in this operator. I think it's a good idea to add those in the fist place to keep the convention.

Comment (kopiczko) [reply service/controller/chart.go]: I missed that discussion. OK, let's then remove that when this change is delivered.

Comment (kopiczko) [reply service/controller/chart.go]: I missed that discussion. But that means we'll have to provide both `Name` and `FinalizerName` to the `ControllerConfig`?

Comment (xh3b4sd) [reply service/controller/chart.go]: Yes.

**Pull request merged**